plugin/src/workletFactory.ts

```diff
@@ -88,7 +87,7 @@ export function makeWorkletFactory(
   // bracket would become part of the comment thus resulting in an error, since
   // there is a missing closing bracket.
   codeObject.code =
-    '(' + (isObjectMethod(fun) ? 'function ' : '') + codeObject.code + '\n)';
+    '(' + (fun.isObjectMethod() ? 'function ' : '') + codeObject.code + '\n)';
```

Comment:
Why do we need these changes in the Babel plugin?

Reply:
Babel has improved its types a bit. Now `isSomething` method can only be applied to nodes, not traversal objects - instead you should use its' methods.